### ISSUE TITLE
number of training data in lmdb released 

### ISSUE THREAD

**Issue by ZENGXH** [ZheC/Realtime_Multi-Person_Pose_Estimation#13]
@ZheC  
Hi Zhe, thanks for the code for the generation of LMDB data. However, I found that there are about 85792 people is training dataset of COCO based on the genlmdb code while there are about 120k sample in the released lmdb. Do I make some mistakes for the counting?

Thanks in advance!

**Comment by ZheC** [comment]
In our lmdb, we have added images from COCO 2014 validation set into training, as in the code (https://github.com/ZheC/Realtime_Multi-Person_Pose_Estimation/blob/master/training/getANNO.m)

There are also other reasons that the number of person in lmdb is not same as the number in COCO dataset: (1) we remove some persons/ images in purpose to create a small validation dataset.  (line 36-39 in https://github.com/ZheC/Realtime_Multi-Person_Pose_Estimation/blob/master/training/genJSON.m) (2) we also skip some persons with very few keypoints or with a tiny scale line 54-56, also persons who are near to others in the same image (to avoid same image occurring too many times in training).

**Comment by ZENGXH** [comment]
Thanks for your explanation. The calculation is consistent now.
